python/xnvme-core/Makefile

```diff
@@ -52,7 +52,7 @@ endef
 .PHONY: install
 install:
 	@echo "## py: make install"
-	@${PY} -m pip install dist/xnvme-*.tar.gz --user
+	@${PY} -m pip install dist/xnvme-*.tar.gz --user --break-system-packages
```

Comment:
There must be a better approach than this. I think the `--break-system-packages` should be avoided if at all possible

Reply:
Agree.
However, the Python offerings need a general re-work/cleanup, thus not spending any more time as this will most like be removed/replaced, and along with that, done "right".